content/_changelogs/12.0.2.md

```diff
@@ -0,0 +1,15 @@
+## 12.0.2
+
+_Released 12/08/2022_
+
+**Bugfixes:**
+
+- Fixed a regression in [12.0.0](#12-0-0), where when `.contains()` receives
+  multiple elements as a subject, it only searched inside the first one. Fixes
+  [#25025](https://github.com/cypress-io/cypress/issues/25025)
+- Fixed a regression in [12.0.0](#12-0-0) around `.contains()` where if the
+  subject was a `<form>` element, it would only consider the first `<input>`
+  element in the form, rather than every child. Fixes
+  [#25019](https://github.com/cypress-io/cypress/issues/25019)
+- Limit the number of globals defined in memory due to the v8 snapshot. Fixed in
+  [#25051](https://github.com/cypress-io/cypress/pull/25051)
```

Comment:
How does this impact the end-user? wondering if we need an entry for this

Reply:
@ryanthemanuel

Reply:
It doesn't really. I'm ok leaving it out.

Reply:
removed